api/main.py

```
@@ -52,7 +53,13 @@ def post(self):
             prediction = randrange(12)
         return {"output":prediction}
 
-API="AIzaSyCZVM0_yjl3Un-mR32EgG1lnFxgQOBNOhE"
+# Load .env file
+load_dotenv()
```

Ruff (0.8.2)

Undefined name `load_dotenv`

(F821)